cmd/fpga_plugin/fpga_plugin.go

```diff
@@ -366,43 +361,25 @@ func main() {
 		debug.Activate()
 	}
 
-	if kubeconfig == "" {
-		config, err = rest.InClusterConfig()
-	} else {
-		config, err = clientcmd.BuildConfigFromFlags(master, kubeconfig)
-	}
+	nodeMode, err := getModeOverrideFromCluster(nodename, kubeconfig, master, mode)
 	if err != nil {
-		fatal(err)
+		fmt.Printf("INFO: could not get mode override from cluster: %+v\n", err)
```

Comment:
I'd suggest to use WARNING here instead of INFO.

Reply:
Sure. I had it WARNING previously but lowered the severity. I can change it back.